src/Form.js

```diff
@@ -0,0 +1,84 @@
+import axios from 'axios';
+import {useState, useEffect} from 'react';
+
+
+const Form = (props) => {
+
+    const [vsCurrencyList, setVsCurrencyList] = useState([]);
+
+    useEffect(() => {
```

Comment:
Consider using a debounce hook to reduce the number of API calls you're making: https://usehooks.com/useDebounce/

Reply:
Having trouble implementing this without it breaking, will readdress it later. The search is not actually making a new API call fortunately. The API call is made on mount, and when the user changes the currency. 

When a user enters a search term, it's just triggering a regular javascript .filter() on the array of data, so not as intense as making an API call each time, but still not necessary to trigger the filter for each individual keystroke.

Need to get a better understanding of debounce before I reattempt, thanks for pointing this out!